site/source/docs/tools_reference/emcc.rst

```diff
@@ -147,6 +149,12 @@ Options that are modified or new in *emcc* are listed below:
   ``-s SEPARATE_DWARF_URL=URL`` to customize that location (this is useful if
   you want to host it on a different server, for example).
 
+.. _emcc-gsplit-dwarf:
+
+``-gsplit-dwarf``
+  Enable debug fission, which creates split DWARF object files alongside the
+  wasm object files. This option must be used together with ``-c``.
```

Comment:
Is this true?  `-gsplit-dwarf` only makes sense when compiling and not when linking?